###################################################################
# Move the added null terminator one char forwards
# dave@slackbuilds.org
###################################################################
diff -Naur a/src/protocol/gopher/gopher.c b/src/protocol/gopher/gopher.c
--- a/src/protocol/gopher/gopher.c	2020-01-08 06:52:23.491455711 +0000
+++ b/src/protocol/gopher/gopher.c	2020-01-08 06:51:41.160210431 +0000
@@ -638,7 +638,7 @@
 	assert(line < end);
 
 	/* Just to be safe NUL terminate the line */
-	end[-2] = 0;
+	end[-1] = 0;
 
 	return line[0] == '.' && !line[1] ? NULL : line;
 }
